processors/scanner.py

```diff
@@ -1,32 +1,75 @@
-from slack_sdk.webhook import WebhookClient
-import logging
+import os, logging
+from datetime import datetime, timedelta
+from processors.github import GitFacade
+from processors.utils import APIUtils
+from slack_sdk import WebClient
+from slack_sdk.errors import SlackApiError
 
 logger = logging.getLogger(__name__)
 
 class Scanner:
 
-    def __init__(self, config, year):
+    def __init__(self, config):
         super().__init__()
         self.config = config
-        self.year = year
+        github_token = os.environ.get("GITHUB_TOKEN")
```

Comment:
Do we need to add this env var to daily-scanner.yml when we run main.py?

Reply:
What about the SLACK_APP_TOKEN?

Reply:
its already available in the github secrets. it might be pulling the environment variables from there.